app/views/constituencies/show.html.haml

```diff
@@ -1,10 +1,11 @@
 #content.section--primary{ tabindex: "0" }
   .container
-    %h1= title(@constituency.name)
-    - @constituency.regions.each do |region|
-      %p.lead
-        = t('.constituency_in')
-        = link_to(region.name, places_show_path(region.gss_code))
+    %h1
+      %span= title(@constituency.name)
+      - @constituency.regions.each do |region|
+        %span.context
```

Comment:
Context shouldn't be indented within the above span

Reply:
It's not; it's in the same level as the above span. It just needs to be indented because of the loop it's in.

Reply:
Ok great